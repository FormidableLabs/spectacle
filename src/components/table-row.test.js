import React from 'react';
import { mount } from 'enzyme';
import TableRow from './table-row';

describe('<TableRow />', () => {
  test('should render correctly', () => {
    const context = {
      styles: {
        components: {
          tableRow: {
            color: '#00f'
          }
        }
      }
    };
    const wrapper = mount(
      <TableRow>
        <td>Table Row Content</td>
      </TableRow>,
      {
        context
      }
    );
    expect(wrapper).toMatchSnapshot();
  });
});
