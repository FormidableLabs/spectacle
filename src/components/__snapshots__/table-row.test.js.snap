// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableRow /> should render correctly 1`] = `
<TableRow>
  <Styled(tr)
    className={undefined}
    style={
      Array [
        Object {
          "color": "#00f",
        },
        Object {},
        undefined,
      ]
    }
  >
    <tr
      className="css-0"
      style={
        Array [
          Object {
            "color": "#00f",
          },
          Object {},
          undefined,
        ]
      }
    >
      <td>
        Table Row Content
      </td>
    </tr>
  </Styled(tr)>
</TableRow>
`;
